Sources/ApodiniDatabase/DatabaseModel.swift

```diff
@@ -0,0 +1,23 @@
+import Fluent
+import Vapor
```

Comment:
Please make sure that Vapor is only implemented as `@_implementationOnly`.
As far as I can see an Foundation import would be sufficient:
```suggestion
import Foundation
```
If you remove the `description` property you can even remove the import.